src/game/test/gameTimerTextBegin.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+"use strict";
+const { jest } = require("@jest/globals");
+const { setTimer } = require("../gameTimer");
+
+const timer = document.createElement("div");
+timer.classList.add("timer");
+document.querySelector("body").append(timer);
+
+const mockFunciton = jest.fn();
```

Comment:
`function` 😄

Reply:
fixed :D i have to always look at this `funciton`